ansible/roles_ocp_workloads/ocp4_workload_rhel_worker/tasks/workload.yml

```diff
@@ -0,0 +1,331 @@
+    iam_role:
+      name: rhel-worker-access-role
+      assume_role_policy_document: "{{ lookup('file','./files/ec2-role-trust-policy.json') }}"
+      create_instance_profile: true
+      description: IAM Role for RHEL Worker trust policy
+      state: present
+        # `create_instance_profile: true` does the following:
+        # aws iam create-instance-profile --instance-profile-name rhel-worker-profile
+        # aws iam add-role-to-instance-profile --instance-profile-name rhel-worker-profile
+        # --role-name rhel-worker-access-role
+
+  - name: Create IAM access policy
+    # vim ec2-role-access-policy.json
+    # aws iam put-role-policy --role-name rhel-worker-access-role \
+    # --policy-name rhel-worker-permissions-policy
+    # --policy-document file://ec2-role-access-policy.json
+    iam_policy:
+      state: present
+      iam_type: role
+      iam_name: rhel-worker-access-role
+      policy_name: rhel-worker-permissions-policy
+      policy_json: "{{ lookup('file', './files/ec2-role-access-policy.json') }}"
+
+  - name: Create RHEL Worker security group
+    register: sg_RHEL_worker
+    ec2_group:
+      region: "{{ aws_region }}"
+      name: "RHEL_worker-{{ cluster_name }}"
+      description: RHEL_worker security group
+      tags:
+        type: RHEL_worker
+      vpc_id: "{{ cluster_vpc }}"
+      rules:
+      - proto: udp
+        ports: 632
+        cidr_ip: 0.0.0.0/0
+      - proto: tcp
+        ports: 22
+        rule_desc: "SSH"
+        cidr_ip: 0.0.0.0/0
+      - proto: udp
+        ports: 5353
+        rule_desc: "mDNS"
+        cidr_ip: 0.0.0.0/0
+      - proto: tcp
+        ports: 80
+        rule_desc: "Ingress HTTP"
+        cidr_ip: 0.0.0.0/0
+      - proto: tcp
+        ports: 443
+        rule_desc: "Ingress HTTPS"
+        cidr_ip: 0.0.0.0/0
+      - proto: tcp
+        ports: 1936
+        rule_desc: "router stats"
+        cidr_ip: 0.0.0.0/0
+      - proto: udp
+        ports: 4789
+        rule_desc: "VXLAN"
+        cidr_ip: 0.0.0.0/0
+      - proto: udp
+        ports: 6081
+        rule_desc: "Geneve"
+        cidr_ip: 0.0.0.0/0
+      - proto: tcp
+        ports: 9000-9999
+        rule_desc: "Worker ingress internal (tcp)"
+        cidr_ip: 0.0.0.0/0
+      - proto: tcp
+        ports: 10250
+        rule_desc: "master ingress kubelet secure"
+        cidr_ip: 0.0.0.0/0
+      - proto: tcp
+        ports: 30000-32767
+        rule_desc: "worker ingress services (tcp)"
+        cidr_ip: 0.0.0.0/0
+      - proto: udp
+        ports: 30000-32767
+        rule_desc: "worker ingress services (udp)"
+        cidr_ip: 0.0.0.0/0
+
+  - name: set a fact for the instance tags
+    set_fact:
+      instance_tags: >-
+        {
+          "type": "RHEL_worker",
+          "guid": "{{ guid }}",
+          "cluster_name": "{{ cluster_name }}",
+          "kubernetes.io/cluster/{{ cluster_name }}": "{{ ocp4_workload_rhel_worker_ownership }}"
+        }
```

Comment:
that's not enough.

You need to merge `hostvars.localhost.cloud_tags_final` on top of this dictionary to have every tags (uuid, platform, etc). Some are mandatory for cost reporting.